crates/runtime/src/dataconnector/github/issues.rs

```diff
@@ -59,6 +59,10 @@ impl GraphQLOptimizer for IssuesTableArgs {
 
         inject_parameters("search", search_inject_parameters, filters, query)
     }
+
+    fn preprocess_value(&self) -> Option<graphql::ValuePreprocessor> {
```

Comment:
Add the same comment from the PR description to this function

Reply:
Can we rely on existing `schema_transform_fn` function?